.github/workflows/release.yaml

```diff
@@ -0,0 +1,92 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
+      - "test-v*"
+
+permissions:
+  contents: read
+
+env:
+  ZIP_FILE: rancher-desktop-networking-${{ github.ref_name }}.zip
+  TAR_FILE: rancher-desktop-networking-${{ github.ref_name }}.tar.gz
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+      - name: create build artifacts
+        run: make build
+        env:
+          CGO_ENABLED: "0"
+      - name: create tarball
+        run: |
+          tar --exclude=*.exe -czf $TAR_FILE ./bin
+          md5sum $TAR_FILE > $TAR_FILE.md5
+      - uses: actions/upload-artifact@v3
+        with:
+          name: rancher-desktop-networking.tar.gz
+          path: rancher-desktop-networking-${{ github.ref_name }}.tar.gz
+          if-no-files-found: error
+      - uses: actions/upload-artifact@v3
+        with:
+          name: rancher-desktop-networking.tar.gz.md5
+          path: rancher-desktop-networking-${{ github.ref_name }}.tar.gz.md5
+          if-no-files-found: error
+      - run: |
+          zip -r $ZIP_FILE ./bin/*.exe
```

Comment:
`zip -r` is for recursing, but we don't have any directories listed?

Reply:
Yes, this is just a flat dir no need for `-r`.